jobbergate-cli/jobbergate_cli/subapps/applications/app.py

```diff
@@ -405,3 +405,89 @@ def download_files(
         ).strip(),
         subject="Application download succeeded",
     )
+
+
+@app.command()
+@handle_abort
+def clone(
+    ctx: typer.Context,
+    id: Optional[int] = typer.Option(
+        None,
+        help=f"The specific id of the application. {ID_NOTE}",
+    ),
+    identifier: Optional[str] = typer.Option(
+        None,
+        help=f"The human-friendly identifier of the application. {IDENTIFIER_NOTE}",
+    ),
+    application_identifier: Optional[str] = typer.Option(
```

Comment:
I wonder if these params should be like `new_application_identifier` and such to make it more clear that these will override the old attributes upon cloning.

Reply:
I considered it, but I chose this path to be consistent with the parameter names we use on `update-application`, and to require less typing from the users. If you agree, I think we'd better keep them in this way.